File: test_bounds.js
const fs = require('fs');
const content = fs.readFileSync('src/components/Dashboard.tsx', 'utf8');
const lines = content.split('\n');

const componentStart = lines.findIndex(l => l.includes('const Dashboard: React.FC'));
const top3 = lines.findIndex(l => l.includes('const top3Bulanan ='));
console.log("Dashboard starts at:", componentStart);
console.log("top3Bulanan starts at:", top3);

File: test_bounds.py
import re

with open('src/components/Dashboard.tsx', 'r') as f:
    lines = f.readlines()

for i, line in enumerate(lines):
    if 'const Dashboard:' in line or 'const Dashboard =' in line:
        print(f"Dashboard starts at: {i}")
    if 'const top3Bulanan =' in line:
        print(f"top3Bulanan starts at: {i}")
    if 'export default Dashboard' in line:
        print(f"export at: {i}")
